src/index.js

```diff
@@ -46,11 +46,23 @@ class Radar {
     return STATUS;
   }
 
-  static initialize(publishableKey) {
+  static initialize(publishableKey, options={}) {
     if (!publishableKey) {
       console.error('Radar "initialize" was called without a publishable key');
     }
     Storage.setItem(Storage.PUBLISHABLE_KEY, publishableKey);
+
+
+    const {cacheLocationMinutes} = options
+
+    if (cacheLocationMinutes) {
```

Comment:
hm we might want an `else` that clears it if `cachedLocationMinutes` is not passed in. in the past, I've run into issues from local storage items getting set and sticking around unintended

for example if you call `initialize` with cachedLocationMinutes=5, then you want to turn it off or on a separate initialize it will stick around even if you don't pass in the field.

Reply:
👍